src/Components/MovieDetailsPage/MovieCast/MovieCast.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from "react";
+import MovieCastCard from "./MovieCastCard";
+import "./moviecast.css";
+
+const MovieCast = ({ id, APIKEY }) => {
+  const [credits, setCredits] = useState({});
+  const cast = Array.from(credits);
+
+  useEffect(() => {
+    const fetchCredits = async () => {
+      const response = await fetch(
+        `https://api.themoviedb.org/3/movie/${id}/credits?api_key=${APIKEY}`
+      );
+      const jsonResponse = await response.json();
+      setCredits(jsonResponse.cast);
+    };
+    fetchCredits();
+  }, []);
```

Comment:
Currently if move id would change, the credits won't be re-fetched, so will be out of date. I would recommend you to add `id` to the dependencies of `useEffect` just to be on the safe side